product-service/handlers/getProductById.ts

```diff
@@ -0,0 +1,44 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import "source-map-support/register";
+// commonjs import for jest importing
+const productList = require("./productList.json");
+
+const headers = {
```

Comment:
Maybe it will be better to move `headers` in separate file? To make it reusable.